src/sql/workbench/browser/modelComponents/media/tabbedPanel.css

```diff
@@ -45,9 +46,10 @@
 }
 
 .tabbedpanel-component .tabList .tab-header.active {
-	background-color: rgb(237, 235, 233);
+	background-color: #E1F0FE;
```

Comment:
how does this look for dark theme?

Reply:
dark theme will not use this color, we have it defined below.